src/autoIncrement.ts

```diff
@@ -92,20 +93,31 @@ export function AutoIncrementID(schema: mongoose.Schema<any>, options: AutoIncre
       return;
     }
 
+    const modelName = (this.constructor as any).modelName;
+    logger.info('modelName "%s"', modelName);
+
     if (!model) {
       logger.info('Creating idtracker model named "%s"', opt.trackerModelName);
       model = this.db.model(opt.trackerModelName, IDSchema, opt.trackerCollection);
+      const counter: AutoIncrementIDTrackerSpec = await model.findOne({ model: modelName, field: opt.field }).lean().exec();
+      if (!counter) {
+        await model.create({
+          model: modelName,
+          field: opt.field,
+          count: opt.startAt > 0 ? opt.startAt - opt.incrementBy : 0
```

Comment:
not allowing negative starting numbers?

Reply:
not. do i have to?

Reply:
i thought your earlier solution was fine, and just set the default `startAt` to `0`, but allow any number

Reply:
done